src/main/kotlin/io/github/mojira/arisa/modules/FutureVersionModule.kt

```diff
@@ -20,6 +20,18 @@ class FutureVersionModule(
                 .filter { it.id in addedVersions }
                 .map { it.remove }
             assertNotEmpty(removeFutureVersions).bind()
+            assertEither(
```

Comment:
If I remember correctly, you can do this in arisa.json instead :D

Reply:
as a note for the future(not needed if you do it in arisa.json), you should add tests to each change

Reply:
oh gosh I don't understand what that means :|